composer.json

```diff
@@ -13,19 +13,17 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
+        "php": ">=8.0",
```

Comment:
Is it possible to keep the php7 compatibility?
```suggestion
        "php": "^7.4|>=8.0",
```

Reply:
I'm using union types to correct deprecations in RedisSessionHandler and it's a php 8 feature 😕

Reply:
You code like a pig 🐷 😜